tests/network-ovn

```diff
@@ -2154,6 +2205,7 @@ else
       echo "Skipping ovn_l3only_tests, not supported"
     fi
     ovn_leases_tests
+    ovn_vlan_uplink_tests
```

Comment:
@boltmark did this test work before https://github.com/canonical/lxd/pull/14196 was merged? 

If not then you're going to need to add an API extension please so that the test can detect support for using a bridge network as an uplink with VLAN.

Suggest `network_ovn_uplink_vlan`

Reply:
Hi @tomponline, the test passes on older lxd versions if we keep

`bridge vlan add dev lxdbr0 vid 10 tagged self`

When it is removed, the test only passes on `latest/edge`. I suppose we should add the API extension then?

Reply:
Yes exactly, thats why we need an API extension check, otherwise you're not testing the functionality in LXD instead the test is just setting up the environment for LXD which defeats the purpose of the test :)

Reply:
Got it, thanks! PR for this here: https://github.com/canonical/lxd/pull/14296.